### PR TITLE
Several minor chef moustache adjustments

### PR DESCRIPTION
**Pull request by LodedDiper69**
## About The Pull Request

Adds a +1 mood boost for chefs wearing their moustache. Adds several new words to the chef moustache vocab, except for "greek" replacing "ops", which got removed.

## Why It's Good For The Game

The mood boost, going along with a trend of similar boosts like clown/clown shoes or sec/black gloves, will encourage chefs to keep wearing their moustache while also not forcing them to. More vocab for the chef is always good, with exceptions like "ops", which more often than not is not used in its intended purpose.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Added a mood boost for chefs wearing their moustaches.
tweak: Reworked the chef moustache vocabulary.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -64,6 +64,17 @@
 			message += pick(" Ravioli, ravioli, give me the formuoli!"," Mamma-mia!"," Mamma-mia! That's a spicy meat-ball!", " La la la la la funiculi funicula!")
 	speech_args[SPEECH_MESSAGE] = trim(message)
 
+/obj/item/clothing/mask/fakemoustache/italian/equipped(mob/user, slot)
+	. = ..()
+	if(slot == ITEM_SLOT_MASK)
+		if(user.mind?.assigned_role == "Cook")
```
Comment (Archanial) [inline code/modules/clothing/masks/miscellaneous.dm]: chain those, no need to nest the code

**Review by Archanial — CHANGES_REQUESTED** [review]

**Comment by Eric-01** [comment]
Is "enemy" supposed to be spoken as "friend-a" and "friend" as "enemy-a"? It doesn't have much to do with this PR, no worries. But since you are on it, I thought I would ask.

File: code/modules/clothing/masks/miscellaneous.dm (hunk 1 above)
Comment (LodedDiper69) [reply code/modules/clothing/masks/miscellaneous.dm]: got it

**Comment by LodedDiper69** [comment]
Honestly not sure myself if it's intended to be swapped (it's an ancient /tg/ relic), but even if it wasn't it's still pretty funny.

**Comment by LodedDiper69** [comment]
Seems to be a problem with the bugfix above. I will close this PR for a moment and investigate what went wrong.

**Pull request closed without merging**